src/Symfony/Component/Workflow/Workflow.php

```diff
@@ -59,10 +59,6 @@ public function getMarking($subject)
     {
         $marking = $this->markingStore->getMarking($subject);
 
-        if (!$marking instanceof Marking) {
```

Comment:
@ro0NL there is no downside into keeping this, please revert this change, let's move on. We won't have a vote on this but on the PR as a whole, as always.